nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -265,6 +271,15 @@
   <section xml:id="sec-release-21.11-incompatibilities">
     <title>Backward Incompatibilities</title>
     <itemizedlist>
+      <listitem>
+        <para>
+          Since <literal>iptables</literal> now use
+          <literal>nf_tables</literal> backend and
+          <literal>ipset</literal> doesn’t support it, some applications
+          (ferm, shorewall, firehol, k3s) may have limited
```

Comment:
What about kubernetes?

Reply:
Kubernetes doesn't have dependency on ipset.

Reply:
How did your K3s tests go? Is this change breaking K3s? (I haven't had the time to test yet.)

Reply:
@euank How do you think this is going to impact K3s?

Reply:
k3s now has proper nftables support: https://github.com/k3s-io/k3s/issues/1812
k3s tests pass:

```bash
$ nix-build --no-out-link .  -A nixosTests.k3s
/nix/store/jdnl8rqh08chrhnnr3gggbbdwaysknkg-vm-test-run-k3s
```